src/main/groovy/net/researchgate/release/ReleasePlugin.groovy

```diff
@@ -262,6 +262,7 @@ class ReleasePlugin extends PluginHelper implements Plugin<Project> {
     }
 
     void updateVersion() {
+        checkPropertiesFile()
```

Comment:
This change was needed as otherwise some of the new tests will fail with:

```
Caused by: org.gradle.api.GradleException: Failed to increase version [unspecified] - unknown pattern
	at net.researchgate.release.ReleasePlugin.updateVersion(ReleasePlugin.groovy:288)
	at groovy.lang.Closure.call(Closure.java:412)
	at groovy.lang.Closure.call(Closure.java:406)
```